.eslintrc.js

```diff
@@ -231,6 +238,35 @@ module.exports = {
                 'no-console': 'off',
             },
         },
+        {
+            // we are using explicit blacklist because this will enforce new rules in newly created packages
```

Comment:
why duplicating this?

Reply:
comment should be updated, I just split the rules to be able to deal just with wrong imports and dependencies and not with the other eslint issues.